src/README.rst

```diff
@@ -16,6 +16,10 @@ To get started, after installation run the following:
 Change Log
 ==========
 
+7.0.3
+----------
+- Add events commands (sfctl events) to retrieve events through EventStore service if installed.
```

Comment:
Please add (#174) at the end of this line and remove the period. Thanks!

Reply:
Sure, I was planning to do so. Thanks for reminding.